client-src/elements/chromedash-gate-chip.js

```diff
@@ -144,9 +148,13 @@ class ChromedashGateChip extends LitElement {
     const stateName = GATE_STATE_TO_NAME[this.gate.state];
     const className = stateName.toLowerCase().replaceAll(' ', '_');
     const iconName = GATE_STATE_TO_ICON[this.gate.state];
-
+    let selectedBorder;
```

Comment:
To avoid the lint error: maybe you could initialize it to an empty string, then you will not need ifDefined().